pyFlowSOM/cyFlowSOM.pyx

```diff
@@ -142,7 +143,7 @@ def som(
     xDists = np.zeros(n_nodes, dtype=np.dtype("d"))
     cdef double [:] xDists_mv = xDists
 
-    if deterministic == True:
+    if seed is not None:
         C_SEED_RAND(2407230991)
```

Comment:
I'm a little bit confused by this here. Wouldn't it be better to set `C_SEED_RAND(seed)` here as well?

It seems like these lines use the value set in C_SEED_RAND to choose which data point to look at.
https://github.com/angelolab/pyFlowSOM/blob/main/pyFlowSOM/flowsom.c#L15-23
https://github.com/angelolab/pyFlowSOM/blob/main/pyFlowSOM/flowsom.c#L8
https://github.com/angelolab/pyFlowSOM/blob/main/pyFlowSOM/flowsom.c#L121

Reply:
My mistake, fixed.